### PR TITLE
added formatting utils

### PR DESCRIPTION
**Pull request by mathieudpnt**
to export aplose results to Raven and json formats

### CODE REVIEW

**Comment by Gautzilla** [comment]
The following part isn't encapsulated in any function: is this intentional? 

```py
# %% export to Raven format

APLOSE_file = Path("path/to/APLOSE/result/file")
df = (
    pd.read_csv(APLOSE_file, parse_dates=["start_datetime", "end_datetime"])
    .sort_values("start_datetime")
    .reset_index(drop=True)
)

df_Raven = APLOSE2Raven(df)


# %% export to json format
df.to_json("output/path.json")

# %% import json format
df_from_json = pd.read_json("path/to/json/file")
```

It looks like a custom use of the `APLOSE2Raven` function. Does it really belong here?

Also (and this is a more overall comment on the OSEkit codebase), the uppercase in `APLOSE` doesn't fit in the PEP styling recommendations: we should decide once and for all of the code we produce whether we intentionally want to go against PEP and stick with APLOSE, or if a lower case `aplose` is acceptable for the project, which would make our code PEP-compliant.

**Comment by mathieudpnt** [comment]
> Also (and this is a more overall comment on the OSEkit codebase), the uppercase in `APLOSE` doesn't fit in the PEP styling recommendations: we should decide once and for all of the code we produce whether we intentionally want to go against PEP and stick with APLOSE, or if a lower case `aplose` is acceptable for the project, which would make our code PEP-compliant.

APLOSE is an acronym (can't recall the meaning though !!) so that's why we write in in CAPS but that's fine to me to write `aplose` to name variables. I would stick to `APLOSE` for comments and text though.


> The following part isn't encapsulated in any function: is this intentional?

I wrote those few lines so a 'beginner' user would be able to use the functions with this example, i'm not sure, should i delete it ?

**Comment by Gautzilla** [comment]
> > Also (and this is a more overall comment on the OSEkit codebase), the uppercase in `APLOSE` doesn't fit in the PEP styling recommendations: we should decide once and for all of the code we produce whether we intentionally want to go against PEP and stick with APLOSE, or if a lower case `aplose` is acceptable for the project, which would make our code PEP-compliant.
> 
> APLOSE is an acronym (can't recall the meaning though !!) so that's why we write in in CAPS but that's fine to me to write `aplose` to name variables. I would stick to `APLOSE` for comments and text though.

I'd say the same thing, lower case in code and upper case in comments and doc

> 
> > The following part isn't encapsulated in any function: is this intentional?
> 
> I wrote those few lines so a 'beginner' user would be able to use the functions with this example, i'm not sure, should i delete it ?

I'd either delete it or add it in the docstring if really needed. The function itself is self-descriptive: it takes a `pandas.DataFrame` as parameter and returns a `pandas.DataFrame`, it is not in the scope of this module how these dataframes should be instantiated.

Maybe, adding to the docstring a quick debunk on what an APLOSE-formatted DataFrame and a Raven-formatted DataFrame contain would do the trick?

File: src/OSmOSE/utils/formatting_utils.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        .reset_index(drop=True)
+    )
+
+    df_raven = aplose2raven(df)
+
+    # export to json format
+    df.to_json("output/path.json")
+
+    # import json format
+    df_from_json = pd.read_json("path/to/json/file")
```
Comment (Gautzilla) [inline src/OSmOSE/utils/formatting_utils.py:30]: Is it related to `aplose2raven`? It is not clear whether `df_from_json` is supposed to be a raven-formatted dataframe or an APLOSE-formatted dataframe.

File: tests/test_utils.py [hunk 2]
```diff
@@ -44,3 +47,51 @@ def test_read_header(input_dir):
...
+    )
+
+    return data.reset_index(drop=True)
+
+
+@pytest.mark.unit
+def test_aplose2raven(correct_dataframe):
+    df_raven = aplose2raven(df=correct_dataframe)
+
+    expected_dataframe = pd.DataFrame(
```
Comment (Gautzilla) [inline tests/test_utils.py:85]: Same here, maybe naming the variables `raven_dataframe` and `expected_raven_dataframe` would make the thing 100% explicit

**Review by Gautzilla — COMMENTED** [review]
I just have very minor remarks on the namings, otherwise I think the test's good!

**Comment by Gautzilla** [comment]
LGTM, I might move the tests later on to a more explicitly-named module, but it'll do the job for now!

**Review by Gautzilla — APPROVED** [review]

**Pull request merged**